cli/src/commands/operation.rs

```diff
@@ -122,6 +141,52 @@ enum UndoWhatToRestore {
     RemoteTracking,
 }
 
+/// Show changes to the repository in an operation
+#[derive(clap::Args, Clone, Debug)]
+pub struct OperationShowArgs {
+    /// Show repository changes in this operation, compared to its parent(s)
+    #[arg(default_value = "@")]
+    operation: String,
```

Comment:
I've been using these (thank you!), but I always want to use `--op` instead of `--operation`. I think it'd be good to add an alias `visible_alias="op"` for `--operation`. (Also for op diff)